### PR TITLE
feat: add moonmail connector

### PR DESCRIPTION
**Pull request by martapa**
## Purpose
- Add MoonmailConnector to send a POST request
- There is a lot of files changed because I ran prettier

## Screenshots
image

## Checklist

- [ ] Title follows the [naming convention](../README.md#pull-requests).
- [ ] Add a description.
- [ ] Add screenshots (optional).
- [ ] Add testing instructions.
- [ ] Add at least one reviewer.
- [ ] Apply related labels (optional).
- [ ] Leave comments on changes you would like to discuss.
- [ ] Remove changes that are not related to this pull request.
- [ ] Remove debugging code.
- [ ] Test your changes locally and on staging (if applicable).
- [ ] Ensure test coverage value in config matches actual coverage (if applicable).
- [ ] Ensure the build is passing.

### CODE REVIEW

File: src/blocks/MoonmailConnector.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import useConnections from './hooks/connections'
+import useProps from './hooks/props'
+
+import { ArgTuple, ComposableProps } from './types'
+
+export const options = {
+  actions: {
+    POSTsuccess: {
```
Comment (mattxyzeth) [inline src/blocks/MoonmailConnector.tsx]: I think prefixing `POST` to these actions is not a great format. I can't think of a use case where these actions are only called from specific requests.

**Review by mattxyzeth — CHANGES_REQUESTED** [review]
Let's update those action names to avoid tech debt.

File: src/blocks/MoonmailConnector.tsx (hunk 1 above)
Comment (karlpawlowicz) [reply src/blocks/MoonmailConnector.tsx]: Fixed, also alphabetized actions to help prevent future merge conflicts

**Review by mattxyzeth — APPROVED** [review]
:shipit:

**Pull request merged**